src/invidious/routes/video_playback.cr

```diff
@@ -131,7 +131,7 @@ module Invidious::Routes::VideoPlayback
       end
 
       # TODO: Record bytes written so we can restart after a chunk fails
-      while true
+      loop do
```

Comment:
Is that change requested by Ameba?
`loop` is just a macro for `while true` anyway:
https://github.com/crystal-lang/crystal/blob/4cea10199d5006000a129413f6f607697185c83a/src/kernel.cr#L114

Reply:
Yep, `Style/WhileTrue`. It does seem like a pretty useless change to me though. 

Should that rule be disabled?

Reply:
Nah, in that case it's good to have a tool that unifies the syntax.